src/lib/pipeline_configuration.py

```diff
@@ -624,6 +630,9 @@ def validate(self):
 
         validators.validate_string(self.memory_profile_upload_url_prefix, "memory_profile_upload_url_prefix")
 
+        if self.listening_group_csv_urls is not None:
+            validators.validate_string(self.pipeline_name, "listening_group_csv_urls")
```

Comment:
I don't think this is right. Shouldn't you be iterating over the listening_group_csv_urls?

Reply:
Yes indeed updated to loop through listening_group_csv_urls list